package/yast2-schema.spec

```diff
@@ -66,7 +66,10 @@ BuildRequires: yast2-ntp-client
 BuildRequires: yast2-online-update-configuration
 BuildRequires: yast2-printer
 BuildRequires: yast2-proxy
+# registration is available only where suse connect is also available
+%ifnarch s390 %ix86
```

Comment:
BTW %ix86 means "i386 i486 i586 i686 pentium3 pentium4 athlon geode", so 32bit

Reply:
yes, I use same think as is used in yast2-registration, so it really match together.